srcpkgs/libglib-static-devel/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'libglib-static-devel'
+#
+# This is only used for qemu-user-static until meson figures out building static
+# glib 2.60.0 via meson.
+#
+pkgname=libglib-static-devel
+version=2.58.3
+revision=3
+wrksrc="glib-${version}"
+build_style=gnu-configure
+configure_args="--enable-libelf --disable-fam --disable-shared --with-pcre=system
+ --enable-static"
+hostmakedepends="automake libtool pkg-config python3 libxslt docbook-xsl"
+makedepends="zlib-devel pcre-devel libffi-devel dbus-devel elfutils-devel libmount-devel"
+depends="$makedepends"
+short_desc="GNU library of C routines"
+maintainer="Rasmus Thomsen <oss@cogitri.dev>"
+license="LGPL-2.1-or-later"
+homepage="https://wiki.gnome.org/Projects/GLib"
+changelog="https://gitlab.gnome.org/GNOME/glib/raw/master/NEWS"
+distfiles="${GNOME_SITE}/glib/${version%.*}/glib-${version}.tar.xz"
+checksum=8f43c31767e88a25da72b52a40f3301fefc49a665b56dc10ee7cc9565cbe7481
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" glib-devel"
```

Comment:
needs to change here as well no ?

Reply:
No, it still needs the glib binaries for cross building